frontend/src/pages/secret-manager/SecretDashboardPage/SecretDashboardPage.tsx

```diff
@@ -26,7 +26,7 @@ import {
   ProjectPermissionDynamicSecretActions,
```

Comment:
IGNORE PLACEMENT OF COMMENT

Can you please run the auto-lint script akhi? I'm seeing this in multiple files 
<img width="821" height="565" alt="image" src="https://github.com/user-attachments/assets/f01f9de1-2907-49ad-8860-6bc7987c99af" />